.github/workflows/release_mac.yml

```diff
@@ -5,6 +5,12 @@ on:
     tags:
       - '*'
   workflow_dispatch:
+    inputs:
+      BUILD_DOCS:
+        description: 'Whether to build docs or not as it takes 15 minutes to install MacTex'
+        required: true
+        type: boolean
+        default: false
```

Comment:
Bonus: on workflow_dispatch, ask if BUILD_DOCS should be enabled or not. Avoids having to tweak the workflow file to turn it off.

Reply:
Beautiful.  Should we do this on all of them?  I could imagine setting up a very generic build workflow where you provide a bunch of switches or CMake args to configure basically the whole build like a normal local setup.  That way we could quickly set a specific configuration with caching/python linking/etc.  Anyway, this is great.